crudd/test.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```

Comment:
The `/tools/` directory has a smattering of tests for things that are both in CI and not in CI.  You could put this there, or leave it here, either way is fine.